agent_api/src/test/java/utils/EmptyAPIResponses.java

```diff
@@ -0,0 +1,25 @@
+package utils;
+
+import dev.aikido.agent_api.background.Endpoint;
+import dev.aikido.agent_api.background.cloud.api.APIResponse;
+import dev.aikido.agent_api.background.cloud.api.ReportingApi;
+import dev.aikido.agent_api.helpers.UnixTimeMS;
+import dev.aikido.agent_api.storage.ServiceConfigStore;
+
+import java.util.List;
+
+import static dev.aikido.agent_api.helpers.UnixTimeMS.getUnixTimeMS;
+
+public class EmptyAPIResponses {
+    public final static APIResponse emptyAPIResponse = new APIResponse(
+            true, "", UnixTimeMS.getUnixTimeMS(), List.of(), List.of(), List.of(), true, false
+    );
+    public final static ReportingApi.APIListsResponse emptyAPIListsResponse = new ReportingApi.APIListsResponse(
+            List.of(), List.of(), ""
+    );
+    public static void setEmptyConfigWithEndpointList(List<Endpoint> endpoints) {
+        ServiceConfigStore.updateFromAPIResponse(new APIResponse(
+                true, "", getUnixTimeMS(), endpoints, List.of(), List.of(), true, false
+        ));
+    }
```

Comment:
My personal opinion is to move these closer to where they are used (e.g.  move `emptyAPIListsResponse` to `APIListsResponse` class as static method or something like that)

Feels like grouping by concept (e.g. like putting all controllers in a controller folder), rather than grouping by functionality/domain

But it's a small one and mostly used for testing so not worth to tackle I guess

Reply:
yeah just easier for testing this way, bc they are almost always imported together